hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/validate/HazelcastSqlValidator.java

```diff
@@ -321,7 +331,6 @@ public Void visit(SqlIdentifier id) {
                             OrderedWindowFinder finder = new OrderedWindowFinder(validator, windowingFunctionFound);
                             SqlNode sqlNode = parseResult.getNode();
                             sqlNode.accept(finder);
-                            // TODO: [sasha] did it as quick idea impl late night, need more thinking about correctness
```

Comment:
I'd say we should remove this whole method. We can't do it correctly, so why bother... The query parsing here is frightening...